### DISCUSSION TITLE
Change the app to use firstname and lastname fields instead of the default name field

### DISCUSSION THREAD

**Discussion by ghost** [munafio/chatify#215] [General]
In my app, I changed the single "name" field to use a firstname and lastname field.  I no longer have the "name" field.

When I click on a user in the search field, the chat app name on the right modal window gets replaced with the name of the person.  How can I change that to use the firstname and lastname fields I have.  Does that have to be done in the javascript?

![2022-09-07_20-07-31](https://user-images.githubusercontent.com/8979815/189007452-41582004-cf05-4109-aa58-8b545f845cda.png)

**Comment by munafio** [comment]
@SiriusPhil yes, in the `code.js` which contains all js code of Chatify .. next to [ line 387](https://github.com/munafio/chatify/blob/29a24b66dcef3966bdabc7a31ac7b6d3f5c0c7ea/src/assets/js/code.js#L387) you can find what you are looking for, change it to yours!
